admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -467,6 +470,20 @@ $product-page-padding-bottom: 80px !default;
     // The not-allowed is actually on the parent "under" the buttons that are disabled
     #product_footer_actions {
       cursor: not-allowed;
+
+      .group-default {
+        display: flex;
+        align-items: center;
+
+        #product_footer_actions_catalog {
+          display: flex;
+          align-items: center;
+
+          .material-icons {
+            margin-top: 0;
+          }
```

Comment:
I really doubt this requires a selector made of 6 components, including 2 IDs 😱

Did you try adding `mt-0` class to that element?

Reply:
I would go for utility classes where we can, maybe not the most performant solution, but it's much safer making changes.

Reply:
hello, ok this is a somewhat specific need (on a single go-to-catalog button). I have reduced the depth of the classes nesting if you prefer.